### ISSUE TITLE
Midpoint Review - Accessibility Feedback - Name of Project

### ISSUE THREAD

**Issue by rsmithadhoc** [department-of-veterans-affairs/va.gov-team#67724]
## VFS actions
- [ ] Assign this ticket to the team member(s) responsible for addressing feedback provided by Platform.
- [ ] Comment on this ticket:
  - [ ] If the Platform reviewer has any **Thoughts/Questions** that require responses.
  - [ ] When **Must** feedback has been incorporated. As appropriate, link to any other GitHub issues or PRs related to this feedback.
  - [ ] When **Should/Consider** feedback has been incorporated, or if any feedback will not be addressed. As appropriate, link to any other GitHub issues or PRs related to this feedback.
- [ ] Close the ticket when all feedback has been addressed.

## Thoughts/questions

-

## Feedback
Practice areas will document their feedback on the VFS-provided artifacts following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback). Platform reviewers may also provide additional notes that don’t comment on the artifacts themselves but are important for implementation (eg. engineering/coding notes).

- **Should:** The research plan mentions confirming screen reader recruitment options, but I would love it if that was more concrete and explicitly called for some assistive technology users.
- **Must:** Design system v1 component styles are used, these should be updated to use [v3 styles](https://design.va.gov/components/#upgrade-to-us-web-design-system-v3) where possible.
- **Must:** I copied this one over from the design intent to reiterate the feedback for PDFs.
    - PDF accessibility! I think I muddled this a little in the meeting, so just to hopefully make it a little clearer:
        - As @artsymartha68 noted, the blank PDF version of the form is generated using tools maintained by a different team and are not your responsibility.
        - But the finished product from your workflow --- writing to a PDF to provide a completed document that can be submitted via mail --- is where you'll need to test for accessibility. Basically, you want to make sure that the document generated by your product is intelligible and navigable for screen reader users, so that they can review their document after completing your workflow. I believe Jeana Clark on the Veteran Facing Forms team may have some useful experience around this, or may be able to point you in the right direction.
- **Must:** Look at simplifying the "Find a Representative" flow and design. If possible, break some of these elements out into their own page to reduce cognitive load. Ideally, implement a pattern of one thing per page. Some suggestions below:
    - If they already have a representative but don't wish to change it, bypass the search for a representative.
    - If you want to change representatives, focus the user on that goal. Hide information and warnings about the current representative.
    - Once you have begun the online form, don't show the PDF download section.
- **Must:** Look at simplifying the "Authorization for Record Access" flow and design. Some suggestions:
    - To follow more of a one thing per page pattern, have the consent question be its own page. If they consent, then show the specific treatments they do not consent to. If they don't consent, bypass the specific treatments question.
    - The modals only contain informational text, consider putting this on the page itself since there is no action being taken in the modal.
- **Must:** Look at using a different component or pattern than a table for the representative search results. Typically, tables should be used for data or relationships, not layout or presentation. It can more difficult to make a table accessible for screen reader users/
- **Must:** Consider using a different component than Additional Info for "Where can I submit my completed Appoint a Representative Form?" due to the amount of content under that. Refer to the [usage guidelines](https://design.va.gov/components/additional-info#usage) for potential options.
    - Headings under step 4 of the process list component appear to be the same heading level 3. If these headings are kept inside the Additional Info component, they should be the next heading level in the hierarchy.
- **Must** - What will the "Change Selected Representative" button do? If an element is navigational (changes the URL in the address bar of the browser), it should be a link. If an element is an action (fires some JavaScript to change content on the page, or saves user information by some mechanism), it should be a button.
- **Must** - What will the "Edit" button on the review page do? If an element is navigational (changes the URL in the address bar of the browser), it should be a link. If an element is an action (fires some JavaScript to change content on the page, or saves user information by some mechanism), it should be a button.
